lib/confusables/generate_list.go

```diff
@@ -47,24 +53,47 @@ func isAllowed(from, to string) bool {
 		}
 	}
 	for _, rn := range toRune {
-		if unicode.In(rn, BasicLatin) || unicode.IsSpace(rn) {
-			return true
+		if !unicode.In(rn, BasicLatin) {
+			return false
 		}
 	}
 
-	return false
+	return true
 }
 
 func formatUnicodeIDs(ids string) string {
 	var formattedIDs string
 	for _, charID := range strings.Split(ids, " ") {
-		newID := fmt.Sprintf("\\U%s%s", strings.Repeat("0", 8-len(charID)), charID)
-		formattedIDs += newID
+		i, err := strconv.ParseInt(charID, 16, 32)
+		if err != nil {
+			fmt.Println(err)
+			continue
+		}
+
+		c := rune(int32(i))
+
+		formattedIDs += string(c)
 	}
 
 	return formattedIDs
 }
 
+// Fixes a lot of issues with the unicode specification, i.e. m -> rn.
+func fixIssuesWithStr(str string) string {
+	// Changes characters such as (16) into 16.
+	parensRegex := regexp.MustCompile(`\((.+)\)`)
+	parensMatches := parensRegex.FindStringSubmatch(str)
+
+	if len(parensMatches) >= 2 {
+		str = parensMatches[1]
+	}
+
+	// Replaces vv and rn with w and m
```

Comment:
Consider making this comment more general or removing it entirely (as is, it's just duplicating the code in `mishapsReplacer`).

Reply:
Agreed